quickstart/201-private-link-sql-database/outputs.tf

```diff
@@ -0,0 +1,28 @@
+output "resource_group_name" {
+  description = "The name of the created resource group."
+  value       = azurerm_resource_group.rg.name
+}
+
+output "virtual_network_name" {
+  description = "The name of the created virtual network."
+  value       = azurerm_virtual_network.my_terraform_network.name
+}
+
+output "subnet_name_1" {
+  description = "The name of the created subnet 1."
+  value       = azurerm_subnet.my_terraform_subnet_1.name
+}
+
+output "nat_gateway"{
+  description = "The name of the created NAT gateway."
```

Comment:
The value seems like nat gateway's id to me, could we rename this output and modify the description?

Reply:
I left that in there because we want the NAT gateway output

Reply:
This output's name is `nat_gateway`, which implies that the value should be `azurerm_nat_gateway.my_nat_gateway`, but the description is `The name of the created NAT gateway`, which implies that the value should be `azurerm_nat_gateway.my_nat_gateway.name`, then we have `azurerm_nat_gateway.my_nat_gateway.id` as the `value`. which one do you want as the output? Could we improve the name because I think it might confuse the reader. WDYT @asudbring?

Reply:
Ahh I understand now.  Thank you for clarifying.  Will review and change and make consistent/clear.
Thanks!

Reply:
I changed it so it is the name that outputs and changed the value to match.  Should clear it up. Ready for workflow kick off.

Thanks!
allen